Dockerfile

```diff
@@ -0,0 +1,21 @@
+# build stage
+FROM node:lts-alpine as build-stage
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm install
+COPY ./ .
+RUN npm run build
+
+
+# production stage
+FROM nginx:stable-alpine as production-stage
+
+LABEL org.opencontainers.image.source=https://github.com/Mosquito-Alert/map
+
+HEALTHCHECK --interval=30s --retries=3 --timeout=5s CMD curl --fail http://localhost || exit 1
```

Comment:
It'd be awesome to set `--start-period=10s --start-interval=3s` to avoid long waiting until the container is health checked. Seems the --start-interval arg is going to be available from Docker engine v.25 (still not released)